src/aggregators/aggregate.js

```diff
@@ -20,6 +20,28 @@
 class Aggregate {
 
   /**
+   * The valid types of aggregation supported by Lighthouse.
+   */
+  static get VALID_TYPES() {
+    return {
```

Comment:
maybe a comment here? If I were writing my own aggregation I wouldn't really understand what the purpose of these types were, if I should add my own, etc